source/common/upstream/load_stats_reporter.cc

```diff
@@ -26,6 +26,7 @@ LoadStatsReporter::LoadStatsReporter(const LocalInfo::LocalInfo& local_info,
 }
 
 void LoadStatsReporter::setRetryTimer() {
+  ENVOY_LOG(info, "Load reporter stats stream/connection will retry in {} ms.", RETRY_DELAY_MS);
```

Comment:
Is there a counter for retry attempts? Otherwise this change will cause a loss of visibility.

Reply:
Currently, the "errors" stat tracks all instances of onRemoteClose, regardless of graceful or not.  Do you think we should add a separate stat for graceful closes specifically, or just continue including the graceful close in the errors stat?  I suppose from a client perspective, a graceful close could be considered to be an error.  I do think the warning log level should be reduced regardless.

Reply:
I think it may be good to add a stat for establishing a new stream. Otherwise it would be difficult to diagnose the  situation where the server keeps closing new streams for some reason.